Index: src/modele/EnemyGrid.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package modele;\r\n\r\n/**\r\n * Class EnemyGrid\r\n */\r\npublic class EnemyGrid extends Grid{\r\n\r\n    public EnemyGrid(){\r\n        this.fill();\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/modele/EnemyGrid.java b/src/modele/EnemyGrid.java
--- a/src/modele/EnemyGrid.java	(revision ad0ea09aa6123333b5f77510e71919b642fbcdf5)
+++ b/src/modele/EnemyGrid.java	(date 1669395551400)
@@ -6,6 +6,6 @@
 public class EnemyGrid extends Grid{
 
     public EnemyGrid(){
-        this.fill();
+        if ()
     }
 }
